app/page.tsx

```diff
@@ -1,3 +1,3 @@
 export default function Page() {
-  return <div>Taskify</div>;
+  return <div className="text-brand-400">Taskify</div>;
```

Comment:
```suggestion
  return <div>Taskify</div>;
```
테스트 코드는 바로 삭제해주시면 됩니다~!